src/app/pages/doc/overview.twig

```diff
@@ -46,9 +45,20 @@
             <ul class="actions">
               {% for docLink in docLinksLE %}
                 <li>
-                  <a href="{{ docLink.url }}" class="button special min-width">
-                    <i class="si si-book"></i> {{ docLink.displayText }}
-                  </a>
+                  <span class="button special min-width">
```

Comment:
why does the LTS version do not have this dropdown?

Reply:
Not done yet. I thought I better wait if you like this change...